.circleci/config.yml

```diff
@@ -1,6 +1,6 @@
 version: 2.1
 
-var_1: &docker_image circleci/node:10.13.0-browsers
+var_1: &docker_image circleci/node:12-browsers
```

Comment:
Is this related to the updated project dependencies or to performance?

Reply:
This is related to the `globalThis` usage in tests. It's supported by all current browsers versions, but Node (used in the tests) just picked it up in v12

Reply:
Moved to `global` on tests and removed this change. We may upgrade it when dApp gets upgraded as well (got some non-sdk related test errors on the dApp on node12).